src/core/stdc/stdio.d

```diff
@@ -939,18 +928,18 @@ else version( CRuntime_DigitalMars )
 else version( CRuntime_Microsoft )
 {
   // No unsafe pointer manipulation.
-  extern (D) @trusted
+  @trusted
```

Comment:
My suggestion is a link time resolution, with the rest of druntime/phobos agnostic to the VS version. So no versions and no imports here.